examples/echo/src/dialer.js

```diff
@@ -17,28 +17,27 @@ async function run() {
   ])
 
   // Dialer
-  const dialerPeerInfo = new PeerInfo(dialerId)
-  dialerPeerInfo.multiaddrs.add('/ip4/0.0.0.0/tcp/0')
   const dialerNode = new Node({
-    peerInfo: dialerPeerInfo
+    addresses: {
+      listen: ['/ip4/0.0.0.0/tcp/0']
+    },
+    peerId: dialerId
   })
 
-  // Peer to Dial (the listener)
-  const listenerPeerInfo = new PeerInfo(listenerId)
-  const listenerMultiaddr = '/ip4/127.0.0.1/tcp/10333/p2p/' +
-      listenerId.toB58String()
-  listenerPeerInfo.multiaddrs.add(listenerMultiaddr)
+  // Add peer to Dial (the listener) into the PeerStore
+  const listenerMultiaddr = '/ip4/127.0.0.1/tcp/10333/p2p/' + listenerId.toB58String()
+  dialerNode.peerStore.addressBook.set(listenerId, [multiaddr(listenerMultiaddr)])
```

Comment:
No need to store this, just dial the multiaddr. Also, multiaddrs shouldn't be stored with the appended peer id